File: C/beginner/1005.c
/*Including Libraries*/
#include <stdio.h>

/*Main Function*/

int main() {
    /*Variable declaration and initialization*/
    float a, b, media;

    /*Requesting values to user*/
    scanf("%f", &a);
    scanf("%f", &b);

    /*Doing the calculations*/
    media = ((a * 3.5) + (b * 7.5)) / 11;

    /*Showing results*/
    printf("MEDIA = %.5f\n", media);

    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/

File: Haskell/beginner/1005.hs
-- Doing imports
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    a <- readLn :: IO Float
    b <- readLn :: IO Float
    
    -- Doing the calculations
    let media = ((a * 3.5) + (b * 7.5)) / 11

    -- Showing results
    printf "MEDIA = %.5f\n" media

-- By Fernando Serra
-- https://github.com/fernandosserra